homeassistant/components/sonos/media_player.py

```diff
@@ -255,6 +221,11 @@ def __init__(self, speaker: SonosSpeaker) -> None:
         self._attr_unique_id = self.soco.uid
         self._attr_name = self.speaker.zone_name
 
+    @property
+    def supported_features(self) -> int:
+        """Return supported media_player features."""
+        return self.speaker.supported_features
```

Comment:
Supported features are not meant to be dynamic

Reply:
For context, this will break everything that reads it at startup (such as voice assistants)